tests/filesystems/test_ffs.py

```diff
@@ -0,0 +1,67 @@
+from datetime import datetime
+from typing import Iterator
+from unittest.mock import Mock, patch
+
+import pytest
+
+from dissect.target.filesystems.ffs import FfsFilesystem, FfsFilesystemEntry
+
+NANOSECONDS_IN_SECOND = 1_000_000_000
+
+
+@pytest.fixture
+def ffs_fs() -> Iterator[FfsFilesystem]:
```

Comment:
I would place this fixture in the `conftest.py` file, rename it to `fs_ffs` and place it at the other `fs_*` fixtures there

Reply:
I wouldn't, it's only used here.

Reply:
@Poeloe The `ffs_fs ` fixture is only used in this test file, so I agree with Schamper that it makes more sense to keep it here. I also kept the fixture in the test file for xfs and jffs2.